test/excel_test.dart

```diff
@@ -8,8 +8,14 @@
 /// * On Excel, go to Insert, My Add-ins (on the ribbon), and select DV360.
 /// * Click on sign-in and allows the web app to access DV360 data.
 /// * Enter an valid advertiserId and insertionOrderId combo.
+/// * Can check/uncheck the highlight underpacing option.
 /// * Click on populate
 ///
 /// Expected Result:
 /// * Should see a new sheet getting created that contains a table of insertion orders.
 /// * Insertion order data values should match with those on the DV360 UI.
+/// * Should see a cell displaying the underpacing threshold value.
+/// * If highlight underpacing is checked, underpacing insertion orders should be
+///   highlighted in red.
+/// * If highlight underpacing is checked, modifying the underpacing threshold value
```

Comment:
is there a way we can test that the correct rows are being highlighted using mocks?
e.g. verify that excel.format(underPace) font was called on rows 1,2,3 but not 4,5,6?

Reply:
lets talk about this

Reply:
lets drop an issue and immediately follow up. while we cant test the office api to excel ui connection, we may be able to test the excelDart to excel api connection.